### ISSUE TITLE
you can't do some of action in offline mode

### ISSUE THREAD

**Issue by Monirzadeh** [DesarrolloAntonio/Shiori-Android-Client#9]
if you in offline mode and try to remove a bookmark you will get error and back to login page
it can be better that it remove from bookmark list in app and if it can't connect to the server keep it in the Queue . next time user connect and has accesses to the server send that delete (or edit or anything else) request to the server.
(maybe an undo option if app not send request yet)

**Comment by Skivling** [comment]
I agree - it should save everything you do and sync with the server once back online

**Comment by DesarrolloAntonio** [comment]
The app already supports full offline mode, saving all actions locally. Pending tasks (e.g., deletions, edits) sync with the server once online. If you're still seeing login errors, share steps to reproduce, or close the issue if resolved. Thanks!
